### PR TITLE
Support for Scheduled Downloads on macOS

### PR DESCRIPTION
**Pull request by FahimF**
These changes add support for scheduled downloading of torrents on a given date/time under macOS by modifying the Add Torrent and Add Magnet views to have a "Start" popup instead of the "Immediately Start Downloading" checkbox.

### CODE REVIEW

File: Transmission.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2272,7 +2271,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "cd third-party/miniupnpc\nln -s . miniupnp";
+			shellScript = "cd third-party/miniupnpc\nrm miniupnp\nln -s . miniupnp";
```
Comment (Coeur) [inline Transmission.xcodeproj/project.pbxproj:2274]: I believe you'd need to revert that change, or explain it

File: Transmission.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2286,7 +2285,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "cd third-party/libutp\nln -s . libutp";
+			shellScript = "cd third-party/libutp\nrm libutp\nln -s . libutp";
```
Comment (Coeur) [inline Transmission.xcodeproj/project.pbxproj:2288]: I believe you'd need to revert that change, or explain it

File: Transmission.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -2300,7 +2299,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "cd third-party/dht\nln -s . dht";
+			shellScript = "cd third-party/dht\nrm dht\nln -s . dht";
```
Comment (Coeur) [inline Transmission.xcodeproj/project.pbxproj:2302]: I believe you'd need to revert that change, or explain it

**Review by Coeur — CHANGES_REQUESTED** [review]

File: Transmission.xcodeproj/project.pbxproj (hunk 2 above)
Comment (Coeur) [reply Transmission.xcodeproj/project.pbxproj:2288]: It was actually already addressed in February 2017 (5 years ago):
https://github.com/transmission/transmission/commit/dcc9a4308b328a87d9c216ccf230faa6844a280b
So this change definitely needs to be removed from this pull request.

File: Transmission.xcodeproj/project.pbxproj (hunk 3 above)
Comment (Coeur) [reply Transmission.xcodeproj/project.pbxproj:2302]: It was actually already addressed in February 2017 (5 years ago):
https://github.com/transmission/transmission/commit/dcc9a4308b328a87d9c216ccf230faa6844a280b
So this change definitely needs to be removed from this pull request.

File: Transmission.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Coeur) [reply Transmission.xcodeproj/project.pbxproj:2274]: It was actually already addressed in February 2017 (5 years ago):
https://github.com/transmission/transmission/commit/dcc9a4308b328a87d9c216ccf230faa6844a280b
So this change definitely needs to be removed from this pull request.

Comment (FahimF) [reply Transmission.xcodeproj/project.pbxproj:2274]: Given that this PR is from Jan 2017 and nothing was done till now, telling me it was addressed in February 2017 might not be the best response? I am no longer working on this after 5 years. Please feel free to close the PR.

File: Transmission.xcodeproj/project.pbxproj (hunk 2 above)
Comment (FahimF) [reply Transmission.xcodeproj/project.pbxproj:2288]: Given that this PR is from Jan 2017 and nothing was done till now, telling me it was addressed in February 2017 might not be the best response? I am no longer working on this after 5 years. Please feel free to close the PR.

File: Transmission.xcodeproj/project.pbxproj (hunk 3 above)
Comment (FahimF) [reply Transmission.xcodeproj/project.pbxproj:2302]: Given that this PR is from Jan 2017 and nothing was done till now, telling me it was addressed in February 2017 might not be the best response? I am no longer working on this after 5 years. Please feel free to close the PR.

Comment (ckerr) [reply Transmission.xcodeproj/project.pbxproj:2302]: @FahimF thanks for the reply.

**Comment by ckerr** [comment]
This PR is pretty old because the project was being neglected for a few years and so the PR didn't get the review and attention it should've gotten when it was submitted. Now, years later, the PR author has understandably moved on and has asked that the PR be closed.

If anyone else comes across this PR because they're looking for this feature, and would like to update it, please do so and @ me for maintainer attention. Thank you!

File: Transmission.xcodeproj/project.pbxproj (hunk 3 above)
Comment (Coeur) [reply Transmission.xcodeproj/project.pbxproj:2302]: My apologies too.
I only started being active in 2022, and there were lots of pending PRs.

**Pull request closed without merging**